assets/assets.go

```diff
@@ -30,8 +32,10 @@ func SeedInitDocs(nd *core.IpfsNode) (*key.Key, error) {
 }
 
 var initDirIndex = []string{
-	filepath.Join("..", "vendor", "dir-index-html-v1.0.0", "knownIcons.txt"),
-	filepath.Join("..", "vendor", "dir-index-html-v1.0.0", "dir-index.html"),
+	// assumes path to this file (assets.go) is:
+	// $GOPATH/src/github.com/ipfs/go-ipfs/assets/assets.go
+	filepath.Join("..", "..", "..", "..", "gx", "ipfs", "QmQfeKxQtBN721pekQh6Jq24adFUjnU65YdY3GNczfuG2T", "dir-index-html", "knownIcons.txt"),
+	filepath.Join("..", "..", "..", "..", "gx", "ipfs", "QmQfeKxQtBN721pekQh6Jq24adFUjnU65YdY3GNczfuG2T", "dir-index-html", "dir-index.html"),
```

Comment:
I know these paths are very hacky, but they're only ever used when running `cd assets/ && go generate` -- I think this is a use case for gx as a library.

```go
import gx "github.com/whyrusleeping/gx-go/lib"

var dirIndexPkg = "/ipfs/QmQfeKxQtBN721pekQh6Jq24adFUjnU65YdY3GNczfuG2T/dir-index-html"
var initDirIndex = []string{
    filepath.join(gx.PathTo(dirIndexPkg), "knownIcons.txt"),
    filepath.join(gx.PathTo(dirIndexPkg), "dir-index.html"),
}
```

Or is there simply a way to get GOPATH? I didn't find one.